test/functional/fixtures/ui/testcafe-fixtures/status-bar-test.js

```diff
@@ -0,0 +1,77 @@
+import { Selector } from 'testcafe';
+
+fixture `Status Bar`
+    .page `http://localhost:3000/fixtures/ui/pages/empty-page.html`;
+
+test('Show status prefix', async t => {
+    const statusDiv = Selector(() => window['%testCafeDriverInstance%'].statusBar.statusDiv);
+
+    let statusText = await statusDiv.innerText;
+
+    await t
+        .expect(statusText).notOk()
+        .expect(statusDiv.innerText).eql('Waiting for assertion execution...');
+
+    await t
+        .eval(() => window['%testCafeDriverInstance%'].statusBar.setStatusPrefix('Status prefix'));
+
+    statusText = await statusDiv.innerText;
+
+    await t
+        .expect(statusText.trim()).eql('Status prefix.')
+        .expect(statusDiv.innerText).eql('Status prefix. Waiting for assertion execution...');
+});
+
+test('Recalculate a view size with a status prefix', async t => {
+    const statusDiv        = Selector(() => window['%testCafeDriverInstance%'].statusBar.statusDiv);
+    const buttons          = Selector(() => window['%testCafeDriverInstance%'].statusBar.buttons);
+    const unlockPageArea   = Selector(() => window['%testCafeDriverInstance%'].statusBar.unlockPageArea);
+    const fixtureContainer = Selector(() => window['%testCafeDriverInstance%'].statusBar.fixtureContainer);
+
+    await t
+        .eval(() => {
+            const statusBar = window['%testCafeDriverInstance%'].statusBar;
+
+            statusBar.setStatusPrefix('Status prefix');
+            statusBar.showDebuggingStatus();
+        });
+
+    await t
+        .resizeWindow(1350, 500);
+
+    const getStatusBarItemsVisibility = async () => {
```

Comment:
IMO it's better to create `checkStatusBarItemsVisibility` method and pass `"expectedObject"` to it. This'll allow us 
1) get rid of properties awaiting. It means we'll use our autowaiting mechanism.
2) compress code little bit

Reply:
I was trying not to await these properties inside assertions, because this may lead to the status text changing to "Wait for element..."